include/dt-bindings/gpio/gpio.h

```diff
@@ -76,6 +76,30 @@
 
 /** @} */
 
+/**
+ * @name GPIO pin voltage flags
+ *
+ * Only applicable if SoC allows to configure pin voltage per individual pin.
```

Comment:
Can you add something like the following to the comment:

`The voltage flags are a Zephyr specific extension of the standard GPIO flags specified by the Linux GPIO binding`

Reply:
Comment adjusted.